test/credit_card_sanitizer_test.rb

```diff
@@ -66,6 +66,11 @@ class CreditCardSanitizerTest < MiniTest::Test
         end
       end
 
+      it "doesn't fail if text is not utf-8 encoded" do
+        ascii_text = '41111111111111112'.force_encoding(Encoding::ASCII)
```

Comment:
I was getting the following error without the new code in the test:

```
  1) Error:
CreditCardSanitizer::#sanitize!#test_0010_doesn't fail if text is not utf-8 encoded:
Encoding::CompatibilityError: incompatible character encodings: US-ASCII and UTF-8
    /Users/ckumar/Code/zendesk/credit_card_sanitizer/lib/credit_card_sanitizer.rb:64:in `scrub!'
    /Users/ckumar/Code/zendesk/credit_card_sanitizer/lib/credit_card_sanitizer.rb:64:in `sanitize!'
    ./test/credit_card_sanitizer_test.rb:71:in `block (3 levels) in <class:CreditCardSanitizerTest>'
```
But I can update the test to check for stripping out the invalid characters